### ISSUE TITLE
a11y: Done & Talk uses a link to submit a classification

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#7276]
## Package
- lib-classifier

## Describe the bug
Done & Talk is styled visually as a button, and behaves like a button. However, it is marked up as a link so screen reader users will hear it announced as a link, and won't expect a link to submit a form.

## To Reproduce
Put yourself in the place of a totally blind volunteer, and use a screen reader to submit a classification for a project workflow.

## Expected behavior
The appropriate semantic HTML element for submitting a form is `<button type=submit >` (or just a plain `<button>`.

## Additional context
[WCAG 1.3.1: Info and relationships](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html) requires us to use semantically appropriate HTML elements for conformance at level A.

Incidentally, sending volunteers to a new tab after submitting a form also fails [WCAG 3.2.5 Change on request](https://www.w3.org/WAI/WCAG22/Understanding/change-on-request.html).
